core/controllers/feedback.py

```diff
@@ -14,6 +14,8 @@
 
 """Controllers for the feedback thread page."""
 
+import feconf
```

Comment:
Actually in this codebase we generally just use the alphabetical order, so I suggest just sticking with that for consistency.